examples/delete_projects.py

```diff
@@ -0,0 +1,42 @@
+'''
+Created on Nov 8, 2019
+
+@author: ylei
+
+Delete multiple projects and their scans. The project names to be deleted are kept in clean_project.json
+'''
+
+import argparse
+import csv
+import logging
+import sys
+import json
+import arrow
+
+from blackduck.HubRestApi import HubInstance
+
+parser = argparse.ArgumentParser("A program that will delete projects along with their scans")
```

Comment:
It's always good to show folks how to delete things, but what is even better is to show how to setup the scans so that they don't have to delete things. The Scanning Best Practices Guide shows how scans can be setup to avoid the need to delete things and still keep the system clean from clutter. Make sure customers are aware of this before pointing recommending they use this script.